pkg/api/util/conditions.go

```diff
@@ -93,7 +99,12 @@ func SetIssuerCondition(i cmapi.GenericIssuer, observedGeneration int64, conditi
 	// If we've not found an existing condition of this type, we simply insert
 	// the new condition into the slice.
 	i.GetStatus().Conditions = append(i.GetStatus().Conditions, newCondition)
-	logf.V(logf.InfoLevel).Infof("Setting lastTransitionTime for Issuer %q condition %q to %v", i.GetObjectMeta().Name, conditionType, nowTime.Time)
+
```

Comment:
```suggestion
```

There seems to be an empty line here. I don't understand why it would be needed.